README.md

```diff
@@ -23,31 +30,66 @@ install path, and that the riscv-gnu-toolchain package is installed.
 The rest of this document describes the format of test programs for the RISC-V
 architecture.
 
+To compile the test, we assume that you have in your `PATH` the directories for
```

Comment:
*tests

Reply:
also (personal note): I don't like the "we" in readmes, I prefer to keep it impersonal :D But if you want to use it, try to stay consistent